tests/integration/test_concurrent_agent_creation.py

```
@@ -0,0 +1,319 @@
+            setup_temp_project(tmp_path)
+
+            # Remove the launcher to cause a failure mid-process
+            (tmp_path / "agents" / "launch").unlink()
+
+            result = run_create_agent(tmp_path, "cleanup-test", "Should fail")
+
+            # Should fail (no launcher)
+            assert result.returncode != 0
+
+            # Agent file should NOT exist (cleaned up)
+            agent_file = tmp_path / ".claude" / "agents" / "cleanup-test.md"
```

Ruff (0.14.14)

[error] Local variable `agent_file` is assigned to but never used

Remove assignment to unused variable `agent_file`

(F841)